tests/partial_span_processor/test_partial_span_processor.py

```diff
@@ -0,0 +1,125 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from time import sleep
+
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.trace import SpanContext, TraceFlags
+
+from src.partial_span_processor import PartialSpanProcessor
+from tests.partial_span_processor.in_memory_log_exporter import \
+  InMemoryLogExporter
+from tests.partial_span_processor.simple_log_record_processor import \
+  SimpleLogRecordProcessor
```

Comment:
why do we need to define our own SimpleLogRecordProcessor?

Reply:
you are right, we do not need it. removed it.